opentracing-api/src/main/java/io/opentracing/Tracer.java

```diff
@@ -18,26 +18,35 @@
 /**
  * Tracer is a simple, thin interface for Span creation and propagation across arbitrary transports.
  */
-public interface Tracer {
+public interface Tracer extends ActiveSpanSource {
 
     /**
      * Return a new SpanBuilder for a Span with the given `operationName`.
      *
+     * <p>If there is an active Span according to the {@link Tracer#activeSpan()}, buildSpan will automatically create
+     * a {@link References#CHILD_OF} reference to that active {@code ActiveSpan.context()}.
+     *
      * <p>You can override the operationName later via {@link Span#setOperationName(String)}.
      *
      * <p>A contrived example:
      * <pre>{@code
```

Comment:
Nit: Nested JavaDoc tags are not supported as far as I know, so I think `<pre>{@code` ... `{@link` does not work and should be changed to `<pre><code>` to make the `{@link` work.